spec/index.html

```diff
@@ -10849,622 +10849,692 @@ <h3>Grammar</h3>
               </tr>
 
               <tr style="vertical-align: baseline">
-                <td><code>[123]&nbsp;&nbsp;</code></td>
+                <td><code>[133]&nbsp;&nbsp;</code></td>
                 <td><code><span class="doc-ref" id="rPrimaryExpression">PrimaryExpression</span></code></td>
                 <td>&nbsp;&nbsp;::=&nbsp;&nbsp;</td>
-                <td><code class="gRuleBody"><a href="#rBrackettedExpression">BrackettedExpression</a> | <a href="#rBuiltInCall">BuiltInCall</a> | <a href="#ririOrFunction">iriOrFunction</a> | <a href="#rRDFLiteral">RDFLiteral</a> | <a href="#rNumericLiteral">NumericLiteral</a> | <a href="#rBooleanLiteral">BooleanLiteral</a> | <a href="#rVar">Var</a> | <a href="#rExprQuotedTriple">ExprQuotedTriple</a></code></td>
+                <td><code class="gRuleBody"><a href="#rBrackettedExpression">BrackettedExpression</a> | <a href="#rBuiltInCall">BuiltInCall</a> | <a href="#ririOrFunction">iriOrFunction</a> | <a href="#rRDFLiteral">RDFLiteral</a> | <a href="#rNumericLiteral">NumericLiteral</a> | <a href="#rBooleanLiteral">BooleanLiteral</a> | <a href="#rVar">Var</a> | <a href="#rExprReifiedTriple">ExprReifiedTriple</a></code></td>
               </tr>
 
               <tr style="vertical-align: baseline">
-                <td><code>[124]&nbsp;&nbsp;</code></td>
+                <td><code>[134]&nbsp;&nbsp;</code></td>
                 <td><code><span class="doc-ref" id="rExprVarOrTerm">ExprVarOrTerm</span></code></td>
                 <td>&nbsp;&nbsp;::=&nbsp;&nbsp;</td>
-                <td><code class="gRuleBody"><a href="#riri">iri</a> | <a href="#rRDFLiteral">RDFLiteral</a> | <a href="#rNumericLiteral">NumericLiteral</a> | <a href="#rBooleanLiteral">BooleanLiteral</a> | <a href="#rVar">Var</a> | <a href="#rExprQuotedTriple">ExprQuotedTriple</a></code></td>
+                <td><code class="gRuleBody"><a href="#riri">iri</a> | <a href="#rRDFLiteral">RDFLiteral</a> | <a href="#rNumericLiteral">NumericLiteral</a> | <a href="#rBooleanLiteral">BooleanLiteral</a> | <a href="#rVar">Var</a> | <a href="#rExprReifiedTriple">ExprReifiedTriple</a></code></td>
               </tr>
 
               <tr style="vertical-align: baseline">
-                <td><code>[125]&nbsp;&nbsp;</code></td>
-                <td><code><span class="doc-ref" id="rExprQuotedTriple">ExprQuotedTriple</span></code></td>
+                <td><code>[135]&nbsp;&nbsp;</code></td>
+                <td><code><span class="doc-ref" id="rExprReifiedTriple">ExprReifiedTriple</span></code></td>
                 <td>&nbsp;&nbsp;::=&nbsp;&nbsp;</td>
                 <td><code class="gRuleBody"><span class="token">'&lt;&lt;'</span> <a href="#rExprVarOrTerm">ExprVarOrTerm</a> <a href="#rVerb">Verb</a> <a href="#rExprVarOrTerm">ExprVarOrTerm</a> <span class="token">'&gt;&gt;'</span></code></td>
```

Comment:
Shouldn't that be a pattern for a triple term now? ..rather than a pattern for a reified triple. In other words, should this be:
```suggestion
                <td><code class="gRuleBody"><a href="#rL_TRIPLE">L_TRIPLE</a> <a href="#rExprVarOrTerm">ExprVarOrTerm</a> <a href="#rVerb">Verb</a> <a href="#rExprVarOrTerm">ExprVarOrTerm</a> <a href="#rR_TRIPLE">R_TRIPLE</a></code></td>
```

If you agree, I suggest to change the name of this production as well; `ExpTripleTerm` may be a good option.

Reply:
Fix applied. The name change is `ExprTripleTerm` ("expr" is used for "expression" in the document elsewhere; "exp" has other meaning like "exponent").

Reply:
> Fix applied. The name change is `ExprTripleTerm` ("expr" is used for "expression" in the document elsewhere; "exp" has other meaning like "exponent").

Looks good! I will mark this branch of the conversation as resolved.

Reply:
Correction made and rule name changed to `ExprTripleTerm` ("expr" issed as an abbreviation of expression elsewhere in the document).